examples/javascript/simple_transfer/transfer.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * This example shows how to use the Aptos client to create accounts, fund them, and transfer between them.
+ */
+
+const { Account, Aptos, AptosConfig, Network } = require("aptos");
+
+const APTOS_COIN = "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>";
+const ALICE_INITIAL_BALANCE = 100_000_000;
+const BOB_INITIAL_BALANCE = 100;
+const TRANSFER_AMOUNT = 0;
+
+/**
+ * Prints the balance of an account
+ * @param aptos
+ * @param name
+ * @param address
+ * @returns {Promise<*>}
+ *
+ * TODO: Change to AccountAddress for address
+ */
+const balance = async (aptos, name, address) => {
+  let balance = await aptos.getAccountResource({ accountAddress: address, resourceType: APTOS_COIN });
+
+  let amount = Number(balance.data.coin.value);
```

Comment:
I was doing this on my PR but got error saying `coin` does not exist in `data`

probably have to cast it? `Number((balance.data as { coin: { value: string } }).coin.value);`

Reply:
Ahh didn't realize this is `js` file. Gotcha